cmd/kwil-admin/cmds/setup/init.go

```diff
@@ -23,49 +28,90 @@ kwil-admin setup init -o ~/.kwild-new`
 )
 
 func initCmd() *cobra.Command {
-	var out, chainId string
+	var out, chainId, genesisPath string
 	var blockInterval time.Duration
 	var joinExpiry int64 // block height
 	var withGas bool
 	var allocs AllocsFlag
+	cfg := config.DefaultConfig()
 
 	cmd := &cobra.Command{
 		Use:     "init",
 		Short:   "The `init` command facilitates quick setup of an isolated Kwil node.",
 		Long:    initLong,
 		Example: initExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			expandedDir, err := common.ExpandPath(out)
+			// TODO: --output-dir and --cfg.root-dir are the same thing, should we remove --output-dir?
+			expandedDir, err := common.ExpandPath(cfg.RootDir)
```

Comment:
Probably cannot just remove it, since it is used: https://github.com/truflation/tsn/blob/8843b2b6841f061787dac3dffa756923fbadfd0e/deployments/infra/lib/kwil-network/generate_genesis.go#L43

Add:
- a deprecation warning
- a TODO that says to remove it for v0.10
- logic that checks if `--output-dir` and cfg.RootDir are set, it errors out.

Reply:
Ah I see you made it deprecated, gj